isthmus/src/main/java/io/substrait/isthmus/IsthmusEntryPoint.java

```diff
@@ -61,20 +68,40 @@ enum OutputFormat {
   // this example implements Callable, so parsing, error handling and handling user
   // requests for usage help or version help can be done with one line of code.
   public static void main(String... args) {
-    int exitCode = new CommandLine(new PlanEntryPoint()).execute(args);
+    logger.debug(Arrays.toString(args));
+    int exitCode = new CommandLine(new IsthmusEntryPoint()).execute(args);
     System.exit(exitCode);
   }
 
   @Override
   public Integer call() throws Exception {
     FeatureBoard featureBoard = buildFeatureBoard();
-    SqlToSubstrait converter = new SqlToSubstrait(featureBoard);
-    Plan plan = converter.execute(sql, createStatements);
-    switch (outputFormat) {
-      case PROTOJSON -> System.out.println(
-          JsonFormat.printer().includingDefaultValueFields().print(plan));
-      case PROTOTEXT -> TextFormat.printer().print(plan, System.out);
-      case BINARY -> plan.writeTo(System.out);
+    // Isthmus image is parsing SQL Expression if that argument is defined
+    if (sqlExpression != null) {
+      logger.debug(sqlExpression);
```

Comment:
should we use these debug logs with some meaningful description without just showing variable values which would be hard to read.

Reply:
Deleted